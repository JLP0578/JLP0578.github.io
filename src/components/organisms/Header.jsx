import React from "react";

require('../../assets/styles/organisms/Header.css');

export default class Header extends React.Component {
    componentDidMount() {
    }

    render() {
        return (
            <header>
                <h1 class="displayNone">Devloppeur Web</h1>
                <i className="state state__low"></i>
            </header>
        );
    }
}