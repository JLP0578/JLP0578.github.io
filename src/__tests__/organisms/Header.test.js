import renderer from 'react-test-renderer';

import Header from '../../components/organisms/Header';

test('renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();

  expect(tree).toMatchSnapshot();

});