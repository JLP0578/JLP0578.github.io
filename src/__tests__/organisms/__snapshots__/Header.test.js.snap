// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <h1
    class="displayNone"
  >
    Devloppeur Web
  </h1>
  <i
    className="state state__low"
  />
</header>
`;
